src/app/forms/templates/hgv/hgv-tech-record.template.ts

```diff
@@ -9,121 +15,150 @@ export const HgvTechRecord: FormNode = {
       name: 'vehicleType',
       label: 'Vehicle type',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.VEHICLETYPE
+      viewType: FormNodeViewTypes.VEHICLETYPE,
```

Comment:
This is out of scope currently as we dont know how this is going to effect the form